src/components/query-search-list/query-search-list.js

```diff
@@ -23,18 +22,17 @@ const QuerySearchList = ({
   scrollable,
   type,
 }) => {
-  const [page, setPage] = useState(FIRST_PAGE);
-
   useEffect(() => {
-    fetch(page);
-  }, [fetch, page]);
+    fetch(FIRST_PAGE);
+  }, [fetch]);
```

Comment:
This can cause issues if reference to `fetch` prop changes. It's better to leave dependencies array empty